### PR TITLE
32436 news media page frontend fixes

### PR DESCRIPTION
**Pull request by lena40**
Steps to test:

1)  Go to /news-media
   a) check page has an intro para
   b) check that the icon boxes do not have a border nor bg on hover

2) Check that promo regions have been added and promo views has been removed from /front and /news-media
     

3) Check that box icons on the front page are themed as previously - with border and bg on hover.
    

4) Check that promo beans on the front page and on /news-media are themed as below and title + image link to its reference entity

6) Check there are borders news-media page as follows:

### CODE REVIEW

File: agov.install [hunk 1]
```diff
@@ -890,6 +890,10 @@ function _agov_default_beans() {
         ),
       ),
     );
+
+
+
+
```
Comment (kimpepper) [inline agov.install]: White space.

Comment (lena40) [inline agov.install]: Fixed. Sorry, thought phpcs would pick that up.

**Comment by kimpepper** [comment]
Spacing for the first icon block looks off.
news-media

**Pull request merged**